### DISCUSSION TITLE
Lilygo T-BEAM V1.1 868MHz SX1262 - CRC mismatch "code -7)

### DISCUSSION THREAD

**Discussion by scytherlein** [jgromes/RadioLib#806] [Q&A]
Hello

we're trying to register the board with otaa in ttn.
Uplink works and we can see the join packets in "live data".
Unfortunately on the board we're always receiving failure code -7 --> CRC mismatch.
TTN Settings are default, LoRa Version 1.1.0

Sketch is the latest "LoRaWAN_End_Device.ino" 
PinConfig changed to `SX1262 radio = new Module(18, 33, 23, 32);`
Keys are edited as well.

With debug enabled we can see "MIC mismatch, expected <hex>, got <other hex>

Can anybody help with this issue or has experienced similar behaviour ?

**Comment by jgromes** [comment]
LoRaWAN supprot is still work-in-progress, at the moment only LoRaWAN 1.0.0 is supported - see #58

**Reply by scytherlein** [reply]
Thank you for the information 👍

**Reply by scytherlein** [reply]
One more question, you said that only V1.0.0 is supported but in the example sketch there is asked for nwkkey (available since V1.1.0).
Is there a way to set nwkkey to "0" and use V1.0.0 instead ?

**Reply by jgromes** [reply]
For v1.0.0, network key is not used. Once v1.1 support is finished, I will clean up the examples to clarify, thanks for the hint.
